tests/test_qaoa_objective_sk.py

```diff
@@ -0,0 +1,187 @@
+###############################################################################
+# // SPDX-License-Identifier: Apache-2.0
+# // Copyright : JP Morgan Chase & Co
+###############################################################################
+import numpy as np
+import pytest
+from functools import partial
+from qiskit_aer import AerSimulator
+from itertools import combinations
+from qokit.sk import sk_obj, get_sk_terms, get_random_J
+from qokit.maxcut import maxcut_obj
+from qokit.qaoa_objective_sk import get_qaoa_sk_objective
+from qokit.utils import brute_force, precompute_energies
+from qokit.parameter_utils import get_sk_gamma_beta
+from qokit.fur import get_available_simulators, get_available_simulator_names
+
+
+rng = np.random.default_rng(seed=42)
+
+qiskit_backend = AerSimulator(method="statevector")
+SIMULATORS = get_available_simulators("x") + get_available_simulators("xyring") + get_available_simulators("xycomplete")
+simulators_to_run_names = get_available_simulator_names("x") + ["qiskit"]
+simulators_to_run_names_no_qiskit = get_available_simulator_names("x")
+
+
+def test_sk_obj(N=5):
+    J = get_random_J(N=N)
+
+    def sk_obj_simple(x, J):
+        obj = 0
+        for i in range(N):
+            for j in range(i, N):
+                obj += J[i, j] * (2 * x[i] - 1) * (2 * x[j] - 1)
+        return 2 * obj / np.sqrt(N)
+
+    x = rng.choice([0, 1], N)
+    assert np.isclose(sk_obj(x, J), sk_obj_simple(x, J))
+
+
+def test_parameter_normalization(N=20):
+    J = get_random_J(N=N)
+
+    f = get_qaoa_sk_objective(N, 1, J=J, parameterization="gamma beta", simulator=simulators_to_run_names[0], objective="expectation")
+    assert np.isclose(f([1], [np.pi / 8]) / N, 0.3, atol=1e-1)
+
+
+def test_variance_normalization(N=1000):
+    J = get_random_J(N=N)
+
+    terms = get_sk_terms(J)
+    terms_0 = [x[0] for x in terms]
+    var = np.var(terms_0)
+    print("var: ", var)
+
+    assert np.isclose(N * var, 1, atol=1e-2)
+
+
+def test_energy_pre_optimized():
```

Comment:
Why not use `@pytest.mark.parametrize("simulator", simulators_to_run_names)` for this test instead of manually iterating over the simulators?

Reply:
Done!